scripts/airdrop.js

```diff
@@ -1,4 +1,6 @@
 const fs = require("fs").promises
+const csv = require("csv-parser")
```

Comment:
Does this require a new package in `package.json` and `yarn.lock`?

Reply:
It does. I am still experimenting with this and probably should have marked it as a draft.